test/collection_test.js

```diff
@@ -2200,14 +2185,14 @@ describe("Collection", () => {
       it("should transfer the headers option", () => {
         return articles.sync({headers: {Foo: "Bar"}})
           .then(() => {
-            expect(pullChanges.firstCall.args[1]).eql({headers: {Foo: "Bar"}});
+            expect(pullChanges.firstCall.args[1]).to.have.property("headers").eql({Foo: "Bar"});
```

Comment:
Ah, I understand, it's because we're adding options to the `pullChanges` calls to track the collection we're syncing against.